src/UI/class-settings-page.php

```diff
@@ -1054,6 +1089,63 @@ public function validate_options( array $input ): array {
 		return $input;
 	}
 
+	/**
+	 * Receive the $input array from the validate_options() function and validate post tracking options.
+	 * This function will mutate the $input array.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @param array $input Array passed to validate_options() function.
+	 * @return void
+	 */
+	private function validate_options_post_type_tracking( array &$input ): void {
+		$options         = $this->parsely->get_options();
+		$posts           = 'track_post_types';
+		$pages           = 'track_page_types';
```

Comment:
Maybe we could pull these two out of the function as constants?

Reply:
I thought of this myself, but in the bigger picture I would consider this as part of #564 as an `options` class or similar which would expose constants for every option key. However, I can do this as an intermediary step if you wish.

Reply:
Fair, we can leave it as is.